openlxp_authentication/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class CoreConfig(AppConfig):
```

Comment:
Generally speaking I like to include the base object name in the child object name to help convey the inheritance context, so in this case the name would be `CoreAppConfig`